packages/apps/graph-client/src/components/chainweb/chainweb-header.tsx

```diff
@@ -0,0 +1,49 @@
+import { selectColor } from '../../utils/selectColor';
+import { Box } from '../box';
+import { Text } from '../text';
+
+import React from 'react';
+
+export function ChainwebHeader(): JSX.Element {
+  return (
+    <Box
+      css={{
+        display: 'flex',
+        alignItems: 'center',
+      }}
+    >
+      <Box css={{ width: '$blockWidth', mx: '$1' }} />
+      {[...new Array(20)].map((_, index) => (
```

Comment:
dont use key as index

Reply:
Normally using the index as a key is an anti-pattern since it can cause more of the component tree to potentially rerender, but in this case I don't think it causes any issues since the array never changes. The array is just being used to create 20 labels using the index. Are you ok with leaving this as is or do you have a recommendation for a better way to implement this?

Reply:
nope if the data is static and there is no removal of items from array causing it to re-render , your solution will do fine:)

Reply:
ur solution is fine (sorry commented from old account)